docker/webui/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN zypper ar -f obs://devel:openQA/openSUSE_Leap_15.2 openQA && \
     zypper ar -f obs://devel:openQA:Leap:15.2/openSUSE_Leap_15.2 openQA-perl-modules && \
     zypper --gpg-auto-import-keys ref && \
     zypper --non-interactive in ca-certificates-mozilla curl && \
-    zypper --non-interactive in --force-resolution openQA apache2 hostname which w3m
+    zypper --non-interactive in --force-resolution openQA apache2 hostname which w3m postgresql
```

Comment:
Better replace the mention of "openQA" with "openQA-local-db", which also pulls in postgresql, see https://github.com/os-autoinst/openQA/blob/master/openQA.spec#L204

Reply:
Done